Install-and-Understand-Byzer-in-3-minutes.md

```diff
@@ -0,0 +1,109 @@
+# Install and Understand Byzer in **3 minutes**
+
+Choose the installation method that works best for your environment:<br>
+Install with **Docker**, **Release Package**, and **Desktop Version**.
+
+We strongly recommend that you use the **Release Package** method, because it can be well oriented to Standalone and Yarn-based distribution. Simply follow 3 steps:<br>
+
+    1. Download
+    2. Unzip and modify the configuration (optional)
+    3. Start service
+
+## Install Byzer Standalone
+### Step1: Download
+    Byzer-lang: https://download.byzer.org/latest/byzer-lang-all-in-one-linux-amd64-3.3.0-2.4.0-SNAPSHOT.tar.gz
+
+    Byzer-notebook: https://download.byzer.org/latest/Byzer-Notebook-1.2.3.tar.gz
+
+### Step2: Unzip
+
+### Step3 Start
+
+Start the Byzer-lang engine
+
+Enter the [byzer-lang-all-in-one-linux-amd64-3.3.0-2.4.0-SNAPSHOT directory](https://download.byzer.org/latest/byzer-lang-all-in-one-linux-amd64-3.3.0-2.4.0-SNAPSHOT.tar.gz), execute the following command to start the Byzer engine:
+```
+./bin/byzer.sh start
+```
+
+After the startup is complete, you can access port 9003 (http://localhost:9003)
+
+To start **Byzer Notebook**, you need to prepare a MySQL database in advance (version 5.7 is recommended), and then create a database named `notebook`.
+Now you can enter `Byzer-Notebook-1.2.3`, and modify the `conf/notebook.properties` file, then modify the database configuration part according to the actual address of the database:
+
+```
+notebook.database.type=mysql
+notebook.database.ip=127.0.0.1
+notebook.database.port=3306
+notebook.database.name=notebook
+notebook.database.username=root
+notebook.database.password=root
+```
+Now it's ready to start **Notebook** by
+
+```
+./bin/notebook.sh start
+```
+
+Visit port 9002 (http://localhost:9002), and enter the **Notebook** interface to start your project.
+
+## Running the Development Server by **Yarn**
+
+### Step 1: 
+Download and Unzip spark-3.3.0: https://archive.apache.org/dist/spark/spark-3.3.0/spark-3.3.0-bin-hadoop3.tgz
+
+### Step 2: 
+Move `core-site.xml`, `yarn-site.xml`, and `hive-site.xml` into the `spark-3.3.0-bin-hadoop3/conf` directory
```

Comment:
Copy  these files from existing Hadoop Cluster to `spark-3.3.0-bin-hadoop3/conf` directory